Octokit.Tests.Conventions/ModelTests.cs

```diff
@@ -97,7 +97,7 @@ public void ResponseModelsHaveReadOnlyCollections(Type modelType)
 
         public static IEnumerable<object[]> GetClientInterfaces()
         {
-            return typeof(IEventsClient)
+            return typeof(GitHubClientTests)
```

Comment:
This should be using something in the `Octokit` project, right? So something like `IGitHubClient`?